src/WorkflowGuiBundle/Controller/WorkflowAdminController.php

```diff
@@ -137,6 +145,17 @@ public function updateAction(Request $request)
         $workflow->setActions($data['actions']);
         $workflow->setTransitionDefinitions($data['transitionDefinitions']);
         $workflow->save();
+        
+        $cfgFile = \Pimcore\Config::locateConfigFile('workflowmanagement.php');
```

Comment:
this is very hacky, why can't you just persist it directly?

Reply:
I see, cause Pimcore would save it as string otherwise right? So you need to manually replace it afterwards.

Reply:
Yes, sir! And I know thats a very ugly hack, but i dont know an other solution :(

If pimcore allows to save classnames instead of ids everthing would be cool, but atm its not possible.